volttron_openadr_ven/agent.py

```diff
@@ -241,24 +241,37 @@ def publish_event(self, event: Event) -> None:
             peer="pubsub",
             topic=f"{topics.OPENADR_EVENT}/{self.ven_client.ven_name}",
             headers={headers.TIMESTAMP: format_timestamp(get_aware_utc_now())},
-            message=OpenADRVenAgent._parse_event(event),
+            message=self._parse_event(event),
         )
 
         return
 
     # ***************** Helper methods ********************
-    @staticmethod
-    def _parse_event(obj: Event) -> Any:
+    def _parse_event(self, obj: Event) -> Any:
         """Parse event so that it properly displays on message bus.
 
         :param obj: The event received from a VTN
         :return: A deserialized Event that is converted into a python object
         """
+
+        # function that gets called for objects that can’t otherwise be serialized.
+        def _default_serialzer(x):
```

Comment:
this should probably be a date or datetime serializer rather than default...as the default is the internal one...I think.

Reply:
Are you suggesting to rename the method to `_datetime_serializer`?